homeassistant/components/homeworks/__init__.py

```diff
@@ -137,12 +144,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     def hw_callback(msg_type: Any, values: Any) -> None:
         """Dispatch state changes."""
         _LOGGER.debug("callback: %s, %s", msg_type, values)
+        if msg_type == HW_LOGIN_INCORRECT:
+            _LOGGER.debug("login incorrect")
```

Comment:
A follow-up PR will initiate a reauth flow here.

Reply:
We should raise ConfigEntryError instead for now, otherwise it keeps retrying

Reply:
This happens if the password is changed while we're connected to the controller, we can't raise then.